rules_building_block/defense_evasion_cmd_copy_binary_contents.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2023/08/23"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Attackers may abuse cmd.exe commands to reassemble binary fragments into a malicious payload.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Binary Content Copy via Cmd.exe"
+risk_score = 21
+rule_id = "53dedd83-1be7-430f-8026-363256395c8b"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+building_block_type = "default"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+  process.name : "cmd.exe" and
+    (process.args : "type" and process.args : ">" or
+     process.args : "type" and process.args : ">>" or
+     process.args : "copy" and process.args : "/b")
```

Comment:
```suggestion
process where host.os.type == "windows" and event.type == "start" and
  process.name : "cmd.exe" and (
    (process.args : "type" and process.args : ">") or
     (process.args : "type" and process.args : ">>") or
     (process.args : "copy" and process.args : "/b"))
```

Reply:
Think these quotes should be added to ensure the logic always functions as intended. Feel free to dispute if that's not the case.

Reply:
Nice catch